pkgs/applications/version-management/forgejo/default.nix

```
@@ -59,12 +65,60 @@ buildGoModule rec {
       --prefix PATH : ${lib.makeBinPath [ bash git gzip openssh ]}
   '';
 
+  passthru.updateScript = writeShellScript "update-forgejo" ''
+    set -o errexit
+    set -o pipefail
+
+    export PATH=${lib.makeBinPath [
+      common-updater-scripts
+      coreutils
+      curl
+      jq
+      nix
+    ]}
+
+    releases=$(mktemp --suffix ".json")
+    curl "https://codeberg.org/api/v1/repos/forgejo/forgejo/releases?draft=false&pre-release=false&limit=1" \
+      --silent \
+      --header "accept: application/json" \
+      --output "$releases"
+
+    stable=$(mktemp --suffix ".json")
+    jq '.[0]
+      | .tag_name[1:] as $version
+      | ("forgejo-src-\($version).tar.gz") as $filename
+      | { $version, html_url } + (.assets | map(select(.name | startswith($filename)) | {(.name | split(".") | last): .browser_download_url}) | add)' \
+      "$releases" >"$stable"
```

this should just drop in, but can we use a variable instead of a tempfile that needs to be cleaned up:
```suggestion
    stable="$(jq '.[0]
      | .tag_name[1:] as $version
      | ("forgejo-src-\($version).tar.gz") as $filename
      | { $version, html_url } + (.assets | map(select(.name | startswith($filename)) | {(.name | split(".") | last): .browser_download_url}) | add)' \
      "$releases")"
```

---

Replaced it with a *here string*, just like with `sha256` :)